crates/re_data_ui/src/component.rs

```diff
@@ -55,13 +55,20 @@ impl DataUi for EntityLatestAtResults {
 
         // Display data time and additional diagnostic information for static components.
         if ui_layout != UiLayout::List {
-            ui.label(format!(
-                "Data time: {}",
-                query
+            let time = self.results.index().0;
+            if time.is_static() {
+                // No need to show anything here. We already tell the user this is a static component elsewhere.
+            } else {
+                let formatted_time = query
                     .timeline()
                     .typ()
-                    .format(self.results.index().0, ctx.app_options.time_zone),
-            ));
+                    .format(time, ctx.app_options.time_zone);
+                let text = format!("Temporal component at {formatted_time}");
+                re_ui::ListItem::new(ctx.re_ui, text)
+                    .with_icon(&re_ui::icons::COMPONENT_TEMPORAL)
+                    .interactive(false)
+                    .show_flat(ui);
+            }
```

Comment:
⚠️ this looks like it might end in a tooltip

Reply:
Not as far as I can find, but I can change it into a horizontal thing instead